src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/index_add_scatter_elements_update.cpp

```diff
@@ -98,7 +98,7 @@ class IndexAddTest : public testing::WithParamInterface<IndexAddTestParams>,
 
         if (ov::element::bf16 == data_type || ov::element::f16 == data_type) {
             configuration.insert({ov::hint::inference_precision.name(), data_type});
-            inType = outType = data_type;
+            outType = data_type;
```

Comment:
In this case we don't need to modify the output type as well, as the operation precision is modified by the `ov::hint::inference_precision`.